openfisca_aotearoa/tests/student_allowance/basic_grant.yaml

```diff
@@ -0,0 +1,53 @@
+---
+
+- name: Student Allowance
+  period: 2018-08
+  absolute_error_margin: 0
+  persons:
+    - id: "fulltime uni student"
+      age: 21
+      is_nz_citizen: true
+      is_tertiary_student: true
+      student_allowance__is_enrolled_fulltime: true
+      student_allowance__meets_attendance_and_performance_requirements: true
+
+
+    - id: "Parttime student"
+      age: 18
+      is_nz_citizen: true
+      is_tertiary_student: true
+      student_allowance__approved_to_study_parttime: true
+
+    - id: "Overseas student"
+      age: 27
+      is_nz_citizen: true
+      is_tertiary_student: true
+      student_allowance__approved_to_study_overseas: true
+
+    - id: "Refugee"
+      age: 99
```

Comment:
Hi, Students over the age of 65 are not eligible for the Student Allowance as age wise they should be eligible for NZ Superannuation - http://www.legislation.govt.nz/regulation/public/1998/0277/latest/DLM260337.html

Reply:
Refugees is still reading as true which it should not be based on age.